NEWS.md

```diff
@@ -45,6 +45,8 @@
 * R >=3.5.0 is now explicitly required. This is in line with the tidyverse
   policy of supporting the [5 most recent versions of
   R](https://www.tidyverse.org/blog/2019/04/r-version-support/).
+  
+* vctrs no longer requires C++ to build, only C (#1815).
```

Comment:
I'd prefer not to mention that in NEWS because it's an implementation detail that doesn't concern users. In almost all cases C++ compilation just works. Also I still think it might be useful in the future for a package like vctrs to call the STL for internal steps of complex algorithms such as those implemented in memtools.